skate-plugin/src/main/kotlin/com/slack/sgp/intellij/tracing/SkateSpanBuilder.kt

```diff
@@ -22,25 +22,23 @@ import com.slack.sgp.tracing.model.newTagBuilder
 class SkateSpanBuilder {
   private val keyValueList: TagBuilder = newTagBuilder()
 
-  fun addSpanTag(key: String, value: String) {
-    keyValueList.apply { key tagTo value }
+  fun addTag(key: String, value: String) {
+    keyValueList.apply { key.lowercase() tagTo value }
   }
 
-  fun addSpanTag(key: String, value: SkateTracingEvent) {
-    keyValueList.apply { key tagTo value.type.name }
+  fun addTag(key: String, event: SkateTracingEvent) {
+    keyValueList.apply { key.lowercase() tagTo event.name }
```

Comment:
Let's always specify a `Locale` argument to these (i.e. `Locale.US`) so we don't have any inconsistencies across different local user locales